### DISCUSSION TITLE
Adding Biome and dprint (rust-based linter & formatter)

### DISCUSSION THREAD

**Discussion by o-az** [numtide/treefmt#262] [Help] [answered]
What would it take to add support for [biome](https://biomejs.dev/) and [dprint](https://dprint.dev/)? I have little experience in rust but I'm determined and I think these two tools are worth it.

Biome is a rust-based linter and formatter that supports: JSONC, JSON, JSX, TypeScript, and JavaScript.
dprint is a rust-based formatter that supports: JSON, Markdown, TOML, Dockerfile, Roslyn, Prettier, Biome, Ruff, TypeScript, JavaScript, Jupyter, rustfmt, yapf, java, and more.

**Comment by zimbatm** [comment, marked as answer]
Here is an example for:
* Biome: https://github.com/numtide/treefmt-nix/blob/main/examples/formatter-biome.toml
* dprint: https://github.com/numtide/treefmt-nix/blob/main/examples/formatter-dprint.toml
